src/hashi.md

```diff
@@ -0,0 +1,13 @@
+# HASHI bridge
+
+This article is about a HASHI bridge - a decentralized two-way bridge connecting SORA and Ethereum. HASHI means bridge in Japanese, and it will be the plumbing that links between many different chains, like pipes connecting the world's assets. This bridge is a significant component of our plans for the beta launch of the SORA network in Q1 2021.
```

Comment:
```suggestion
This article is about a HASHI bridge - a decentralized two-way bridge connecting SORA and Ethereum. HASHI means bridge in Japanese, and it will be the plumbing that links many different chains, like pipes connecting the world's assets. This bridge was a significant component of the plans for the beta launch of the SORA network in Q1 2021.
```